server/resources/network.go

```diff
@@ -46,6 +49,9 @@ type EpochStart struct {
 
 // AggregatedNodeStatus is an aggregated resource
 type AggregatedNodeStatus struct {
+	Version                        string
+	ConnectedPeersCounts           map[string]int
```

Comment:
why do we need to process these connected peer counts? It is strictly for informational use?

Reply:
Strictly informational (debugging), indeed.